app/server.go

```diff
@@ -727,11 +735,27 @@ func runSecurityJob(s *Server) {
 	}, time.Hour*4)
 }
 
+func doDiagnosticsIfNeeded(s *Server) {
+	hoursSinceFirstServerRun := time.Since(time.Unix(*s.Config().ServiceSettings.ServerFirstRun, 0)).Hours()
+	// Send once every 10 minutes for the first hour
```

Comment:
I'm worried that admins might be surprised that data is send to a third party so quickly after setting up a server.

In the past there have been a lot of reports by community members complaining about our aggressive collecting of data. (https://github.com/mattermost/mattermost-server/issues/10595, https://github.com/mattermost/mattermost-server/issues/9466) With this changes, Admins who care about there data, won't be able to disable telemetry in time.

cc @adamjclarkson

Reply:
@adamjclarkson , @grundleborg  thoughts?

Reply:
cc @wiersgallak

Reply:
The analytics we are collecting are not changing, it's only the frequency that we're collecting them. So there is no encroachment here on privacy. Admins are still able to turn off telemetry and opt out at any time. The data itself gives us clarity on what Admins are doing during the first 24 hours in order to improve the setup process, guiding admins more in the future through configuration.

Reply:
The discussion isn't really about privacy, is about data ownership. Surely admins can opt out, but at the point they data already has been send, which is a bummer for some people. 

There are a lot of reports from admins who didn't know telemetry exist and discover to late that data has been send. With this change the amount of data that has been send will be even higher.

It's also unlikely that an admin will find out in the first 10 minutes that telemetry exists. Hence, this will surprise a lot of admins.

Reply:
I get it. From my end we desperately need more data to help us understand what Admins are doing (or not doing) in the first 24 hours to help us understand why 75% of servers don't last longer than 1 day. Improving 7 day server retention is one of the key objectives under our V2MOM and this data will provide critical information as we try to solve that problem. 
I am currently working on another project that is a redesign of the admin onboarding flow. In it we could introduce an opt-out for telemetry making clearer to users that we are collecting, link to the documentation and provide easy way for them to decline.  No surprises.

Reply:
It great to hear that the on boarding experience is planed to be redesigned.  :+1: 

With this in mind I have less concerns with merging this PR also because I see the need to improve the 7 day server retention.